README.md

```diff
@@ -66,3 +66,25 @@ const mySchema = new Schema({
   marks
 });
 ```
+### `Yalc`: An alternative to yarn link - Testing with flexible-content locally
```

Comment:
```suggestion
### Testing locally in applications using `prosemirror-elements`
```

I've reworded here to make this slightly more generic, both because I don't think it matters that we happen to be using `yalc`, and because some users may not know (or care!) about `flexible-content` – bear in mind that this repo is designed to be agnostic!